core/proto/keytransparency_v1_types/keytransparency_v1_types.proto

```diff
@@ -41,9 +41,8 @@ message Committed {
 
 // EntryUpdate contains the user entry update(s).
 message EntryUpdate {
-  // update authorizes the change to entry.
-  // TODO(gdbelvin): rename to mutation.
-  SignedKV update = 2;
+  // mutation authorizes the change to entry.
+  SignedKV mutation = 2;
```

Comment:
You might want to consider using a new tag ID for this.
Although field renaming is compatible on the wire if you have any old text-serialized protos to be compatible with you'll break when the new code reads them.

Reply:
This rename is merely for developer clarity. It maintains the same functionality. 
We don't have any text serialized protos that I know of.